maintainers/scripts/haskell/update-head.hackage.hs

```diff
@@ -0,0 +1,211 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -p "haskellPackages.ghcWithPackages (p: with p; [aeson optparse-applicative])"
+#! nix-shell -p git
+#! nix-shell -p nix-prefetch-git
+#! nix-shell -p nixfmt
+#! nix-shell -i runhaskell
+
+
+{-
+
+This is largely copied from the generate-nix-overrides.hs script in the
+head.hackage repo.
```

Comment:
What license is that? Having an [SPDX Id](https://spdx.dev/ids/) in the comment here would be good if necessary

Reply:
yeah, I've actually already asked on the ghc channel as there wasn't one in the repo. Won't merge without adding it